crates/daphne/src/vdaf/mastic.rs

```diff
@@ -149,13 +149,13 @@ pub(crate) fn mastic_prep_finish_from_shares(
             // valid. This is not secure because the weight is revealed to the caller.
             let peer_weight = Field64::get_decoded(peer_share_bytes)?;
             if peer_weight != host_weight {
-                return Err(VdafError::Vdaf(prio::vdaf::VdafError::Uncategorized(
+                return Err(VdafError::Vdaf(PrioVdafError::Uncategorized(
```

Comment:
Revert the change. By the way, we included the `prio::vdaf::` prefix here so that we didn't have to rename this type. I generally try to avoid `use foo::Bar as FooBar` pattern because it creates a degree of indirection that makes the code harder to read.